extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -571,18 +577,32 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     if (this.useSra && this.element.getAttribute(DATA_ATTR_NAME)) {
       user().warn(TAG, 'Cannot enable a single slot for both refresh and SRA.');
     }
-    this.refreshManager_ = this.useSra ||
-        getEnclosingContainerTypes(this.element).filter(container =>
-            container != ValidAdContainerTypes['AMP-CAROUSEL'] &&
-            container != ValidAdContainerTypes['AMP-STICKY-AD']).length
-            ? null
-            : this.refreshManager_ || new RefreshManager(this, {
-              visiblePercentageMin: 50,
-              continuousTimeMin: 1,
-            });
+    this.initializeRefreshManagerIfEligible_();
     return superReturnValue;
   }
 
+  /**
+   * Will initialize a refresh manager for this slot if this slot is
+   * refresh-enabled and is not part of an SRA set. If a refresh manager is
+   * already initialized for this slot, it will be reused.
+   * @param {number=} refreshInterval The refresh interval in seconds. If
+   *    passed, will forcefully enable refresh on this slot, regardless of
+   *    initial page configuration (e.g., meta tags and slot attributes).
+   * @private
+   */
+  initializeRefreshManagerIfEligible_(refreshInterval) {
+    this.refreshManager_ = this.refreshManager_;
```

Comment:
Huh?  This should be:

if (!refreshInterval || this.refreshManager_ || this.useSra || getEnclosingTypes(...)) {
   return;
}
this.refreshManager_ = new RefreshManager(this, {
        visiblePercentageMin: 50,
        continuousTimeMin: 1,
      }, refreshInterval);